Packs/RecordedFuture/Integrations/RecordedFutureLists/RecordedFutureLists_description.md

```diff
@@ -0,0 +1,26 @@
+## Information  
+A valid API Token for XSOAR from Recorded Future needed to fetch information.
+[Get help with Recorded Future for Cortex XSOAR](https://www.recordedfuture.com/support/demisto-integration/).  
+
+**Version:** 1.0.0
+
+---
+
+## Configuration
+| Parameter                                      | Description                                                                                                                                                                                                                                 |
+|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Server URL                                     | The URL to the Recorded Future ConnectAPI.                                                                                                                                                                                                  |
+| API Token                                      | Valid API Token from Recorded Future.                                                                                                                                                                                                       |
+| Classifier                                     | Select "Recorded Future Playbook Alert Classifier".                                                                                                                                                                                                      |
+| Mapper (Incoming)                              | Select "Recorded Future Playbook Alert Mapper".                                                                                                                                                                                                                                                                                                              |
```

Comment:
Does the Classifier and Mapper of "Recorded Future Playbook Alerts" suit this as well?
If so, why do you think the new commands need to be in a new pack?

Reply:
Missed removing this, will do. We have decided to have these as different apps to not clutter the specific integrations for the clients since they can use the apps very differently.

Reply:
done